docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,178 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(masternode_announce_message)
+  
+**Result**
+
+  A message with the result of the broadcasting, it could be a success message or an error one including the reason of failure.
+
+masternode.subscribe
+--------------
+
+Returns the status of masternode.
+
+**Signature**
+
+  .. function:: masternode.subscribe(masternode_collateral)
```

Comment:
Parameter must match the code.  How about "collateral" or just "address" (I'm guessing but I assume that's what it is).  Please document the parameter.